website/tsconfig.json

```diff
@@ -1,13 +1,18 @@
 {
-  "extends": ["../tsconfig.options.json", "@docusaurus/tsconfig"],
+  "extends": [
+    "../tsconfig.options.json",
+    "@docusaurus/tsconfig"
+  ],
   "compilerOptions": {
     "emitDeclarationOnly": true,
     "noEmit": false,
     "outDir": "../.moon/cache/types/website",
     "verbatimModuleSyntax": false,
     "baseUrl": ".",
     "paths": {
-      "@site/*": ["./*"]
+      "@site/*": [
+        "./*"
+      ]
```

Comment:
I didn't want to change this, but prettier was quite adamant that it didn't like its previous state, so I wanted to minimize the chances that the CI pipeline would fail.